system_metrics_collector/test/system_metrics_collector/test_linux_memory_measurement.cpp

```diff
@@ -46,6 +46,7 @@ namespace
 {
 constexpr const char kTestNodeName[] = "test_measure_linux_memory";
 constexpr const char kTestMetricName[] = "system_memory_percent_used";
+constexpr const char kTestMetricUnit[] = "percent";
```

Comment:
Nit: this is used more than once throughout the tests, why not define in test constants?

Reply:
How about just using what is in `constants.hpp` instead of adding another copy to `test_constant.hpp`?